02-1.py

```diff
@@ -0,0 +1,17 @@
+from utils import read_inputs
+
+inputs = read_inputs('inputs/input2.txt')
+
+right_pswd = 0
+
+for ele in inputs:
```

Comment:
I prefer the term `line` rather than `ele`. Both term describe vague entities, but at list `line` is not a shortcut.  What do you think ?